### ISSUE TITLE
Run the fixed training RL script and check the logits difference

### ISSUE THREAD

**Issue by Barahlush** [NelsonG-C/lasr-labs-2025-control-project#66]

**Comment by Barahlush** [comment]
Fixed the reward, running again.
Countdown + password:
https://wandb.ai/baraltiva-individual/rl-self-conditioning/runs/bozjzbxa?nw=nwuserbaraltiva

Countdown vanilla:
https://wandb.ai/baraltiva-individual/huggingface/runs/2lcdx6q0?nw=nwuserbaraltiva

it kind of worked (task reward grew up), but ideally it needs either 1) simplify the task; 2) higher reward for correct task solution; 3) more training steps; 4) providing formatting reward only when solution is correct (Joan recommendation). Since first ~300 steps it was optimizing only for formatting, and only last 200 steps gradually optimized for solution correctness.

**Comment by NelsonG-C** [comment]
@Barahlush Did we see increases in the logits or any affect on outputs when using the fixed model?

**Comment by Barahlush** [comment]
Logits didn't change, results are on slides:
https://docs.google.com/presentation/d/1ZoH8xMt1A2t6N_WU43ysqPCoKZPrCnd5IZbR-WGs9ZU/edit?slide=id.g348a0695688_0_1#slide=id.g348a0695688_0_1

And training runs are here:
https://wandb.ai/baraltiva-individual/rl-self-conditioning/workspace?nw=nwuserbaraltiva

One idea from Jonathan on why it failed: since the password was provided in the prompt, RL probably reinforced just the copying circuits, not ones directly related to the password tokens.
